11_async.md

```diff
@@ -199,44 +110,24 @@ bigOak.readStorage("food caches", caches => {
 });
 ```
 
-(All binding names and strings have been translated from crow language
-to English.)
+(所有的绑定名和字符串已经从乌鸦语被翻译成英语了。)
 
-This style of programming is workable, but the indentation level
-increases with each asynchronous action because you end up in another
-function. Doing more complicated things, such as running multiple actions
-at the same time, can get a little awkward.
+这种编程风格是行得通的，然而缩进的层级会随着每一次异步操作而增加，因为你终将处于另一个函数中。如果做更为复杂的事情，比如同时运行多项操作，可能会变得有些尴尬。
 
-Crow nest computers are built to communicate using
-((request))-((response)) pairs. That means one nest sends a message to
-another nest, which then immediately sends a message back, confirming
-receipt and possibly including a reply to a question asked in the
-message.
+鸦巢计算机是为了以((请求))-((响应))模式来进行通信而建造的。这意味着一个鸟巢对另一个鸟巢发送一条讯息，另一个鸟巢立即发回一条确认收到、有可能还包含对之前讯息中的问题的回复的讯息。
 
-Each message is tagged with a _type_, which determines how it is
-handled. Our code can define handlers for specific request types, and
-when such a request comes in, the handler is called to produce a
-response.
+每条讯息都被标上了一个 _type(译者注：类型)_，以决定该讯息是如何被处理的。我们的代码可以定义特殊的请求类型的处理器，当这个类型的请求出现时，相应的处理器会被调用，从而生成一个响应。
 
 {{index "crow-tech module", "send method"}}
 
-The interface exported by the `"./crow-tech"` module provides
-callback-based functions for communication. Nests have a `send` method
-that sends off a request. It expects the name of the target nest, the
-type of the request, and the content of the request as its first three
-arguments, and it expects a function to call when a response comes in as its
-fourth and last argument.
+`"./crow-tech"` 模块导出的接口提供了基于回调的函数来进行通信。鸟巢拥有一个可以发送请求的 `send` 方法。该方法预期一个目标鸟巢的名称、请求的类型，以及请求的内容作为它的前三个参数，还预期一个在收到响应时就调用的函数作为其第四个(也是最后一个)参数。
 
 ```
 bigOak.send("Cow Pasture", "note", "Let's caw loudly at 7PM",
             () => console.log("Note delivered."));
```

Comment:
discussion: should we translate the strings here?

Reply:
If we translate the code block, would it affect other language versions when it got merged? If other versions would be intact, then I reckon we could it.

Reply:
why would it affect other languages? my understanding is this branch is specific to chinese

Reply:
Ok, I will translate the text inside the code block then.